app/views/subscriptions/forms/_subscription_fields.html.erb

```diff
@@ -3,7 +3,7 @@
     <%= label_tag('subscription[Name]', 'Subscription Name', class: 'eui-required-o') %>
   </div>
   <div class="row">
-    <%= text_area_tag('subscription[Name]', @subscription['Name'], cols: 30, rows: 3, class: 'required textcounter', maxlength: 255) %>
+    <%= text_field_tag('subscription[Name]', @subscription['Name'], cols: 30, rows: 3, class: 'required full-width textcounter', maxlength: 80) %>
```

Comment:
is there not a test for this? 3 humans have now had to deal with it.

Reply:
The best I can do now is update the test to use a name that is 80 characters long.

Reply:
well, then lets just think about this then, and you go forward as is.